conf.d/plugin-bang-bang.fish

```diff
@@ -0,0 +1,16 @@
+bind ! __history_previous_command
+bind '$' __history_previous_command_arguments
+
+# set up the same key bindings for insert mode if using fish_vi_key_bindings
+if test "$fish_key_bindings" = 'fish_vi_key_bindings'
+    bind --mode insert ! __history_previous_command
+    bind --mode insert '$' __history_previous_command_arguments
+end
+
+function _plugin-bang-bang_uninstall --on-event plugin-bang-bang_uninstall
+    bind --erase --all !
+    bind --erase --all '$'
+    echo "plugin-bang-bang key bindings removed"
+
```

Comment:
How about removing this line? Implementation details are implementation details :-)
```suggestion
```